frontend/src/lib/auth/permissions.ts

```diff
@@ -5,39 +5,47 @@ import {
 import { User } from "../api/auth";
 import { DestructionList } from "../api/destructionLists";
 
+export type PermissionCheck = (user: User) => boolean;
+
+export type DestructionListPermissionCheck = (
+  user: User,
+  destructionList: DestructionList,
+) => boolean;
+
+export const canChangeSettings: PermissionCheck = (user) => {
+  // TODO: Functioneel beheerder in future
+  return user.role.canStartDestruction;
```

Comment:
This relates to another issue right?

Reply:
This is already in for the settings page, I refactored the file